modules/src/ping.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+
+from templates.text import TextTemplate
+
+def process(input, entitites):
+    output = {}
+    try:
+        url = entities['url'][0]['value']
+        orig_url = url
+        strip_list = ['http://','https://','/']
+        for item in strip_list:
+            url = url.strip(item)
+        page = requests.get('https://isitup.org/'+url+'.json')
+        page = json.loads(page.text)
```

Comment:
If you see the other modules, the general structure is:
r = requests.get(...)
data = r.json()